packages/components/src/resize-tooltip/styles/resize-tooltip.styles.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * External dependencies
+ */
+import styled from '@emotion/styled';
+
+/**
+ * Internal dependencies
+ */
+import Text from '../../text';
+import { color } from '../../utils/style-mixins';
+
+export const Root = styled.div`
+	bottom: 0;
+	box-sizing: border-box;
+	left: 0;
+	pointer-events: none;
+	position: absolute;
+	right: 0;
+	top: 0;
+`;
+
+export const TooltipWrapper = styled.div`
+	align-items: center;
+	box-sizing: border-box;
+	display: inline-flex;
+	justify-content: center;
+	opacity: 0;
+	pointer-events: none;
+	transition: opacity 120ms linear;
+
+	${ ( { isActive } ) =>
+		isActive &&
+		`
+	opacity: 1;
+` }
```

Comment:
We should indent these lines to make it clear they are part of the condition.

Reply:
This is how my VSCode auto-formatted it (with Prettier and ESLint).
I admit... it doesn't look very nice. I'll rewrite it another way 🙌 

Thanks for the note!